### PR TITLE
[2.10] Latest tests security analytics

### PR DESCRIPTION
**Pull request by amsiglan**
### Description
Latest tests in security analytics plugin

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: cypress/utils/plugins/security-analytics-dashboards-plugin/index.d.ts
Comment (ruanyl) [inline cypress/utils/plugins/security-analytics-dashboards-plugin/index.d.ts:1]: Just curious why adding this file?

Comment (amsiglan) [reply cypress/utils/plugins/security-analytics-dashboards-plugin/index.d.ts:1]: Had it in the plugin repo to help with the types

Comment (ruanyl) [reply cypress/utils/plugins/security-analytics-dashboards-plugin/index.d.ts:1]: Oh I see, those are for custom cypress commands, thanks!

**Comment by amsiglan** [comment]
@ruanyl Any idea why the tests are taking so long to run?
Also, can you please review the changes, need them for the 2.10 release

**Comment by ruanyl** [comment]
@amsiglan should this backport to `main`?

**Review by ruanyl — APPROVED** [review]

**Review by peterzhuamazon — APPROVED** [review]

**Pull request merged**